website/settings/defaults.py

```diff
@@ -129,64 +68,18 @@ def parent_dir(path):
 # External services
 USE_CDN_FOR_CLIENT_LIBS = True
 
-USE_EMAIL = True
-FROM_EMAIL = 'openscienceframework-noreply@osf.io'
-
-# support email
-OSF_SUPPORT_EMAIL = 'support@osf.io'
-
-# prereg email
-PREREG_EMAIL = 'prereg@cos.io'
-
 # Default settings for fake email address generation
 FAKE_EMAIL_NAME = 'freddiemercury'
 FAKE_EMAIL_DOMAIN = 'cos.io'
 
-# SMTP Settings
-MAIL_SERVER = 'smtp.sendgrid.net'
-MAIL_USERNAME = 'osf-smtp'
-MAIL_PASSWORD = ''  # Set this in local.py
-
-# OR, if using Sendgrid's API
-# WARNING: If `SENDGRID_WHITELIST_MODE` is True,
-# `tasks.send_email` would only email recipients included in `SENDGRID_EMAIL_WHITELIST`
-SENDGRID_API_KEY = None
-SENDGRID_WHITELIST_MODE = False
-SENDGRID_EMAIL_WHITELIST = []
-
-# Mailchimp
-MAILCHIMP_API_KEY = None
-MAILCHIMP_WEBHOOK_SECRET_KEY = 'CHANGEME'  # OSF secret key to ensure webhook is secure
-ENABLE_EMAIL_SUBSCRIPTIONS = True
-MAILCHIMP_GENERAL_LIST = 'Open Science Framework General'
-
-#Triggered emails
-OSF_HELP_LIST = 'Open Science Framework Help'
-PREREG_AGE_LIMIT = timedelta(weeks=12)
-PREREG_WAIT_TIME = timedelta(weeks=2)
-WAIT_BETWEEN_MAILS = timedelta(days=7)
-NO_ADDON_WAIT_TIME = timedelta(weeks=8)
-NO_LOGIN_WAIT_TIME = timedelta(weeks=4)
-WELCOME_OSF4M_WAIT_TIME = timedelta(weeks=2)
-NO_LOGIN_OSF4M_WAIT_TIME = timedelta(weeks=6)
-NEW_PUBLIC_PROJECT_WAIT_TIME = timedelta(hours=24)
-WELCOME_OSF4M_WAIT_TIME_GRACE = timedelta(days=12)
-
 # TODO: Override in local.py
 MAILGUN_API_KEY = None
 
-# Use Celery for file rendering
-USE_CELERY = True
-
 # File rendering timeout (in ms)
 MFR_TIMEOUT = 30000
 
-# TODO: Override in local.py in production
-DB_HOST = 'localhost'
```

Comment:
To avoid confusion with the app's Postgres settings, we should rename these to something like `SHAREJS_DB_HOST`, etc.

I've made a follow-up ticket: https://openscience.atlassian.net/browse/PLAT-676